### ISSUE TITLE
HeaderGlobalActions and HeaderMenuButton missing aria attributes

### ISSUE THREAD

**Issue by lee-chase** [carbon-design-system/carbon#2973]
HeaderGlobalActions and HeaderMenuButton should have aria-controls and aria-haspopup attributes. The aria-controls attribute should be set to the id of the panel it is controlling.

## What package(s) are you using?
components and react as a reference to build @carbon/vue

## Steps to reproduce the issue

1. View HTML source on the storybook site
 http://react.carbondesignsystem.com/?selectedKind=UI%20Shell&selectedStory=Header%20Base%20w%2F%20Actions%20and%20Right%20Panel&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel.

**Comment by dakahn** [comment]
Closing this one down. After some research the support and behavior we see for `aria-controls` isn't the experience we're going for here. Instead properly using landmarks and having the expandable area immediately following the `aria-expanded` controller has more consistent cross platform screen reader support -- which is our target 

https://heydonworks.com/article/aria-controls-is-poop/
